### PR TITLE
HDS-1637: Add new troubleshooting item into site README

### PR DESCRIPTION
**Pull request by minevala**
## Description
- Add new troubleshooting item into the documentation site README

## Related Issue
- https://helsinkisolutionoffice.atlassian.net/browse/HDS-1637

## Motivation and Context
- Better repository documentation makes it easier to develop HDS which saves time and therefore 💰

### CODE REVIEW

File: site/README.md [hunk 1]
```diff
@@ -74,3 +74,8 @@ PATH_PREFIX='/hds-demo/docsite-fixes' yarn build -- --prefix-paths
 ## Troubleshooting
 - **Site build fails with error #11903 COMPILATION or something similar**. Gatsby's cache might not be in sync with the source files due to a checkout (chancing branches, etc.)  
   **Solution:** Run `$(npm bin)/gatsby clean` or manually remove .cache folder to clean up cache files. Then try to run the build again.
+
+- **Development: A new local component is not available when trying to import from the hds-react in the site**:
+  1. Review that all component exports exist in hds-react package
```
Comment (harriplappalainen) [inline site/README.md:79]: Should we mention specifically the component and components folder's index.ts files so there's no ambiguity?

Comment (NikoHelle) [reply site/README.md:79]: How about adding a script for this to the <root>/package.json

`"rebuild-site": "yarn --cwd site clean && yarn build && yarn --cwd site start"`

I guess `lerna run` could also be used, but I'm not familiar with that.

Comment (NikoHelle) [reply site/README.md:79]: Also requirement to add new css to all.css is not documented.

Comment (minevala) [reply site/README.md:79]: I will add a link to development.md

Comment (minevala) [reply site/README.md:79]: The exports are explained in here: https://github.com/City-of-Helsinki/helsinki-design-system/blob/8f9342a8a2a5a1fdaad35025fa2e82cf5c698065/packages/react/DEVELOPMENT.md

**Review by harriplappalainen — APPROVED** [review]
Good, good 👍

**Pull request merged**